src/WebApi/appsettings.Development.json

```diff
@@ -4,5 +4,8 @@
       "Default": "Information",
       "Microsoft.AspNetCore": "Warning"
     }
+  },
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=localhost;Database=CleanTodos;User Id=<<whatever-your-db-username-is>>;Password=<<whatever-your-stong-password-is>>;TrustServerCertificate=true"
```

Comment:
`appsettings.json` states the connection string should be configured in user secrets.  Which way do we want to go here?